sdk/databox/Microsoft.Azure.Management.DataBox/src/Generated/Models/ShippingAddress.cs

```diff
@@ -32,17 +32,17 @@ public ShippingAddress()
         /// </summary>
         /// <param name="streetAddress1">Street Address line 1.</param>
         /// <param name="country">Name of the Country.</param>
-        /// <param name="postalCode">Postal code.</param>
         /// <param name="streetAddress2">Street Address line 2.</param>
         /// <param name="streetAddress3">Street Address line 3.</param>
         /// <param name="city">Name of the City.</param>
         /// <param name="stateOrProvince">Name of the State or
         /// Province.</param>
+        /// <param name="postalCode">Postal code.</param>
         /// <param name="zipExtendedCode">Extended Zip Code.</param>
         /// <param name="companyName">Name of the company.</param>
         /// <param name="addressType">Type of address. Possible values include:
         /// 'None', 'Residential', 'Commercial'</param>
-        public ShippingAddress(string streetAddress1, string country, string postalCode, string streetAddress2 = default(string), string streetAddress3 = default(string), string city = default(string), string stateOrProvince = default(string), string zipExtendedCode = default(string), string companyName = default(string), AddressType? addressType = default(AddressType?))
```

Comment:
This looks like a  breaking change due to making a parameter optional.  Please add a customization with a constructor with the same parameter order as the previous version (since this is a partial class, you can just implement the backwards-compatible constructor).

Reply:
@dhawal777 You are right that adding a new constructor won't fix this.  The simplest solution is likely to update the csharp readme to make the postal code required again for c#.